File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/openTripMap/SimpleSuggestFeature.java
package com.tammamkhalaf.myuaeguide.categories.openTripMap;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SimpleSuggestFeature {

    @SerializedName("xid")
    @Expose
    private String xid;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("highlighted_name")
    @Expose
    private String highlightedName;
    @SerializedName("dist")
    @Expose
    private Double dist;
    @SerializedName("rate")
    @Expose
    private Integer rate;
    @SerializedName("osm")
    @Expose
    private String osm;
    @SerializedName("kinds")
    @Expose
    private String kinds;
    @SerializedName("point")
    @Expose
    private Point point;
    @SerializedName("wikidata")
    @Expose
    private String wikidata;

    public String getXid() {
        return xid;
    }

    public void setXid(String xid) {
        this.xid = xid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getHighlightedName() {
        return highlightedName;
    }

    public void setHighlightedName(String highlightedName) {
        this.highlightedName = highlightedName;
    }

    public Double getDist() {
        return dist;
    }

    public void setDist(Double dist) {
        this.dist = dist;
    }

    public Integer getRate() {
        return rate;
    }

    public void setRate(Integer rate) {
        this.rate = rate;
    }

    public String getOsm() {
        return osm;
    }

    public void setOsm(String osm) {
        this.osm = osm;
    }

    public String getKinds() {
        return kinds;
    }

    public void setKinds(String kinds) {
        this.kinds = kinds;
    }

    public Point getPoint() {
        return point;
    }

    public void setPoint(Point point) {
        this.point = point;
    }

    public String getWikidata() {
        return wikidata;
    }

    public void setWikidata(String wikidata) {
        this.wikidata = wikidata;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/openTripMap/SimpleSuggestFeature.kt
package com.tammamkhalaf.myuaeguide.categories.openTripMap

import com.google.gson.annotations.Expose

class SimpleSuggestFeature {
    @SerializedName("xid")
    @Expose
    private var xid: String? = null

    @SerializedName("name")
    @Expose
    private var name: String? = null

    @SerializedName("highlighted_name")
    @Expose
    private var highlightedName: String? = null

    @SerializedName("dist")
    @Expose
    private var dist: Double? = null

    @SerializedName("rate")
    @Expose
    private var rate: Int? = null

    @SerializedName("osm")
    @Expose
    private var osm: String? = null

    @SerializedName("kinds")
    @Expose
    private var kinds: String? = null

    @SerializedName("point")
    @Expose
    private var point: com.tammamkhalaf.myuaeguide.categories.openTripMap.Point? = null

    @SerializedName("wikidata")
    @Expose
    private var wikidata: String? = null
    fun getXid(): String? {
        return xid
    }

    fun setXid(xid: String?) {
        this.xid = xid
    }

    fun getName(): String? {
        return name
    }

    fun setName(name: String?) {
        this.name = name
    }

    fun getHighlightedName(): String? {
        return highlightedName
    }

    fun setHighlightedName(highlightedName: String?) {
        this.highlightedName = highlightedName
    }

    fun getDist(): Double? {
        return dist
    }

    fun setDist(dist: Double?) {
        this.dist = dist
    }

    fun getRate(): Int? {
        return rate
    }

    fun setRate(rate: Int?) {
        this.rate = rate
    }

    fun getOsm(): String? {
        return osm
    }

    fun setOsm(osm: String?) {
        this.osm = osm
    }

    fun getKinds(): String? {
        return kinds
    }

    fun setKinds(kinds: String?) {
        this.kinds = kinds
    }

    fun getPoint(): com.tammamkhalaf.myuaeguide.categories.openTripMap.Point? {
        return point
    }

    fun setPoint(point: com.tammamkhalaf.myuaeguide.categories.openTripMap.Point?) {
        this.point = point
    }

    fun getWikidata(): String? {
        return wikidata
    }

    fun setWikidata(wikidata: String?) {
        this.wikidata = wikidata
    }
}